deployment/src/strongmind_deployment/acm.py

```diff
@@ -0,0 +1,105 @@
+import re
+from pydantic import BaseModel
+from pulumi import Output, ResourceOptions
+from pulumi_aws import acm, route53
+import pulumi
+
+
+class AcmCertificateArgs(BaseModel):
+    stack: str
+    zone_id: str
+    cert_fqdn: str = None
+    tags: dict = None
+    domain: str = "aws2.strongmind.com"
+
+
+class AcmCertificate(pulumi.ComponentResource):
+    def __init__(
+        self, name: str, args: AcmCertificateArgs, opts: pulumi.ResourceOptions = None
+    ):
+        super().__init__(
+            "strongmind:global_build:commons:acmcertificate", name, None, opts
+        )
```

Comment:
to keep the look and feel similar,   lets set primitive property types in the constructor directly, but use a `create_resources()` function that will create the certificate resources here.